cegs_portal/search/templates/search/v1/experiment_list.html

```diff
@@ -243,6 +242,31 @@ <h2 class="accordion-header mb-0" id="headingThree">
     addDragListeners();
     addSelectListeners();
 </script>
+<script>
+    window.addEventListener('DOMContentLoaded', (event) => {
+        window.addEventListener('resize', function() {
+            if (window.innerWidth <= 768) {
+                document.querySelector('#headingOne button').setAttribute('aria-expanded', 'false');
+                document.querySelector('#headingTwo button').setAttribute('aria-expanded', 'false');
+                document.querySelector('#headingOne button').setAttribute('data-te-collapse-collapsed', '');
+                document.querySelector('#headingTwo button').setAttribute('data-te-collapse-collapsed', '');
+                document.querySelector('#collapseOne').classList.add('hidden');
+                document.querySelector('#collapseTwo').classList.add('hidden');
+                document.querySelector('#collapseOne').removeAttribute('data-te-collapse-show');
+                document.querySelector('#collapseTwo').removeAttribute('data-te-collapse-show');
+            } else {
+                document.querySelector('#headingOne button').setAttribute('aria-expanded', 'true');
+                document.querySelector('#headingTwo button').setAttribute('aria-expanded', 'true');
+                document.querySelector('#headingOne button').removeAttribute('data-te-collapse-collapsed');
+                document.querySelector('#headingTwo button').removeAttribute('data-te-collapse-collapsed');
+                document.querySelector('#collapseOne').classList.remove('hidden');
+                document.querySelector('#collapseTwo').classList.remove('hidden');
+                document.querySelector('#collapseOne').setAttribute('data-te-collapse-show', '');
+                document.querySelector('#collapseTwo').setAttribute('data-te-collapse-show', '');
```

Comment:
"headingOne"/"headingTwo" and "collapseOne"/"collapseTwo" are not very good names -- they aren't descriptive enough. Can you improve them?